.github/jenkinsfile/recipeUpdater.groovy

```diff
@@ -0,0 +1,7 @@
+gitPullRequestPipeline {
+	dockerImageName = 'symbolplatform/build-ci:cpp-ubuntu-lts'
+	branchName = 'fix/update_catapult_recipe'
```

Comment:
i guess should be changed to `main` before merge?

Reply:
This is the pull request branch name. I will rename it for clarity.
The default base branch, ``main`` for this repo, is configured in the Jenkins job and can be overridden when the job is run manually.